### DISCUSSION TITLE
React-Native Compatibility?

### DISCUSSION THREAD

**Discussion by SamRiles** [jaames/iro.js#242] [General]
I've been attempting to implement iro in an expo go react-native app. I havent been able to find much about implementation in react-native, but was curious if iro is compatible with react-native? 

Are there any easily accessible examples to the implementation? 

Thanks!

**Comment by jaames** [comment]
It's not compatible with react native unfortunately. I'm not super familiar with it, but as I understand there aren't a lot of fleshed out options for SVG / canvas style rendering in react native (at least not ones that rely on a webview), so implementation would be non-trivial.

**Reply by SamRiles** [reply]
Thanks for the quick reply and insight. That saves me some time. Appreciate it!
